src/ParseServer.js

```diff
@@ -64,73 +64,80 @@ class ParseServer {
     const {
       appId = requiredParameter('You must provide an appId!'),
       masterKey = requiredParameter('You must provide a masterKey!'),
-      cloud,
-      security,
       javascriptKey,
       serverURL = requiredParameter('You must provide a serverURL!'),
-      serverStartComplete,
-      schema,
     } = options;
     // Initialize the node client SDK automatically
     Parse.initialize(appId, javascriptKey || 'unused', masterKey);
     Parse.serverURL = serverURL;
 
     const allControllers = controllers.getControllers(options);
-
-    const {
-      loggerController,
-      databaseController,
-      hooksController,
-      liveQueryController,
-    } = allControllers;
+    options.state = 'initialized';
```

Comment:
Could you explain the new states of Parse Server and when they apply?

Reply:
`initialized`: server has been created but `.start` has not been called yet
`starting`: server is starting, database connecting etc
`error`: there was a startup error
`ok`: server started ok and is running

Reply:
I've added a section to the README